roadmaps/2023roadmap.md

```diff
@@ -0,0 +1,127 @@
+# OpenMDAO Development Map 2023
+  
+Author: Rob Falck  
+Date: 2023-02-16 
+  
+## What is this roadmap for?  
+As in previous years, this roadmap exists to provide general guidance for our development decisions in the coming year, and to provide a retrospective for our performance in the previous year.
+  
+## Performance Assessment for 2022 
+
+As a general platform for the development of multidisciplinary analysis and optimization tools,OpenMDAO achieved significant uptake in its usage at NASA.
+The agency is building its next generation of aircraft conceptual design tools on an OpenMDAO foundation.
+High fidelity multi-physics analysis tools are being developed to work together in an OpenMDAO ecosystem.
+OpenMDAO will have a significant role to play in the next generation of the agency's digital engineering strategy.
+
+We were pleased to host the first in-person OpenMDAO workshop since 2019.
+This provided the community with a look at how various organizations are using OpenMDAO, and provided valuable feedback for us as developers.
+We get to see ways in which the tool has enabled some really impressive work by the community.
+We also get to learn about challenges the community faces and develop ways to overcome these challenges.
+
+It's fair to say that OpenMDAO has reached a level of maturity.
+The fundamental capability is sound, and through some iteration we've arrived at a stable API.
+With a strong, technically sound foundation much of our focus in the next year will be on continuing to increase the productivity of users by removing friction where we can.
+
+- Our goal of developing an interactive training course has produced a great series of [Practical MDO](https://www.youtube.com/@OpenMDAO/featured) videos and corresponding notebooks.
+- Work on the [Mphys](https://github.com/OpenMDAO/mphys) package has pushed our capability in terms of large models involving parallel computing. This work resulted in a [significant API change](https://github.com/OpenMDAO/POEMs/blob/master/POEM_075.md) to make the use of distributed components more clear, with some more improvements on the way.
+- We continue to move away from reliance on feedback through the terminal and instead provide richer feedback through reports. We've received positive feedback on the standard set of reports we generate and will continue to improve upon these and develop new ones when warranted.
+- The push to bring 2nd-derivative information into the Modular Approach to Unified Derivatives (MAUD) theory upon which OpenMDAO is based remains an open goal but there is still interest in that area.
+ 
+## 2023 Focus Areas 
+
+A broad goal for OpenMDAO is to make things simple for ~80% of our users who are doing typical analysis, while making things possible for the ~20% of our users who are advanced.
```

Comment:
Do 80% and 20% come from somewhere?

Reply:
Only from a general [80/20 rule](https://en.wikipedia.org/wiki/Pareto_principle)